rfcs/text/0000/host.yml

```diff
@@ -0,0 +1,25 @@
+- name: host
+  fields:
+    - name: model
```

Comment:
Capturing these types of inventory attributes has come up in past ECS discussions. One pitfall to avoid would be limiting them to certain field sets that wouldn't allow them to describe a broader range of assets someone might have in their inventory or CMDB.

Examples could be power supplies, generators, or server racks. These items would still have `model`, `manufacturer`, or `serial_number` attributes to capture but wouldn't necessarily still be considered hosts in the ECS sense of a host.

In past brainstorming, the idea of creating an `inventory.*` or `asset.*` field set has been suggested, but I think that idea would best be discussed as its own RFC.

Reply:
I am not sure I understand the distinction you're trying to make here.  We have other lower level objects that are disimmilar (.name).  Not sure why using a .model for host would preclude using a .model with a different description for another object would be problematic?  The inventory. or asset. concept is still talking about an entity (a host), so it would be nice for context but it would also make my search problematic (E.G. if I needed to see a SW inventory on a host).  In that case the fact that the scan came from tenable or my EDR host module would be the indicator that it was a point in time inventory vice an event with an associated host.

Reply:
I think what @ebeahan was trying to say was that in ECS, we describe host broadly as a 'general computing instance' meaning it can be anything from hardware to virtual machine to docker container, etc. The intention with a `inventory.*` or `asset.*` would be specifically a physical item we want to keep track of.

Reply:
As mentioned in https://github.com/elastic/ecs/pull/1512#issuecomment-952973774, I like the symmetry of using `product` over `model` to match the existing `observer.product` field naming.